code/modules/surgery/generic.dm

```diff
@@ -140,104 +125,40 @@
 		self_msg = "<span class='warning'> Your hand slips, damaging several organs [target]'s lower abdomen with \the [tool]!</span>"
 	user.visible_message(msg, self_msg)
 	target.apply_damage(12, BRUTE, affected, sharp = 1)
-	return 0
+	return FALSE
 
 /datum/surgery_step/generic/cauterize
-
+	priority = -1 // Always on the bottom of possibilities
```

Comment:
Todo actually use the priority value. Maybe to fix the available steps issue with say IMS